README-MLservices.md

```diff
@@ -0,0 +1,169 @@
+# Graymeta Machine Learning services
+This is optional but is required if you want to use any Graymeta Machine Learning services.
+
+**NOTE:** *The first module below is required to setup a internal load balancer to the different ML services.*
+
+**UPGRADE:** *If upgrading Faces from v0.1.4 or lower make sure you copy the `RDS Configuration` to the new module and that the module does not change.  Also credits and slates was included with the old faces module.  Now they are seperate module and make sure you update the endpoint/hostname in the extractor config.*
```

Comment:
s/credits and slates was/credits and slates were/
s/now they are separate module/now they are separate modules/

what does it mean that the "module does not change"?

Reply:
Let me rephrase s/module does not change/module name does not change/.
As long as the module name does not change the RDS will not have to be destroyed and recreated.